test/heavymock/heavymock_test.go

```diff
@@ -52,7 +54,7 @@ func TestConnect(t *testing.T) {
 			t.Fatalf("%v.Export(_) = _, %v", client, err)
 		}
 		require.NoError(t, err, "Err listening stream")
-		require.True(t, SimpleRecord.Equal(record), "Incorrect response message")
+		require.Equal(t, SimpleRecord.Record.ID.Pulse()+10, record.Record.ID.Pulse(), "Incorrect response message")
```

Comment:
I don't understand why record PN should be greater.  we were sending request field and expecting the same record.

Reply:
the magic is we do not send a record here, it is empty pulse, finished by record from next pulse.